frontend/components/FlashMessage/_styles.scss

```
@@ -17,7 +17,9 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  color: $core-fleet-white;
+  // Use static (un-themed) white: the flash toast is always a colored
```

Stylelint (17.7.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)